pages/linux/cmus.md

```diff
@@ -2,14 +2,19 @@
 
 > Commandline Music Player.
 > Use arrow keys to navigate, `<enter/return>` to select, and numbers 1-8 switch between different views.
+> More information: <https://cmus.github.io/>.
 
-- Open cmus from specified directory:
+- Add file/directory to library:
```

Comment:
The first example should always invoke the command. Otherwise, one wouldn't know where to enter the commands.